### PR TITLE
[tests] Fix sample tests to take into account when a platform has been disabled.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
✅ [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-9774/1/display/redirect) (on internal Jenkins) succeeded

✅ [Provisioning succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/1//console)
✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/1//console)
✅ Packages built successfully
<details><summary>View packages</summary>

* [xamarin.ios-14.7.0.44.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/xamarin.ios-14.7.0.44.pkg)
* [Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg)
* [Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg)
* [Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1167+sha.5645f5c85.nupkg)
* [Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)
* [Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/package/Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.543+sha.5645f5c85.nupkg)

</details>

✅ [API Diff (from stable)](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/jenkins-results/api-diff/index.html)
✅ [API Diff (from PR only)](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/jenkins-results/apicomparison/api-diff.html) (no change)
ℹ️ [Generator Diff](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/5645f5c8574cdb4bd6d6f3168006777dd930350d/1/jenkins-results/generator-diff/index.html) (please review changes)
ℹ️ Test run skipped: Not running tests here because they're run on public Jenkins.

**Comment by xamarin-release-manager** [comment]
✅ [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-9774/2/display/redirect) (on internal Jenkins) succeeded

✅ [Provisioning succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/2//console)
✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/2//console)
✅ Packages built successfully
<details><summary>View packages</summary>

* [xamarin.ios-14.7.0.45.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/xamarin.ios-14.7.0.45.pkg)
* [Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg)
* [Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg)
* [Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1168+sha.e5d4e7c10.nupkg)
* [Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)
* [Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/package/Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.544+sha.e5d4e7c10.nupkg)

</details>

✅ [API Diff (from stable)](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/jenkins-results/api-diff/index.html)
✅ [API Diff (from PR only)](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/jenkins-results/apicomparison/api-diff.html) (no change)
ℹ️ [Generator Diff](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/e5d4e7c10fa75920f819ed9bfbac94888fa5b198/2/jenkins-results/generator-diff/index.html) (please review changes)
ℹ️ Test run skipped: Not running tests here because they're run on public Jenkins.

File: msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx [hunk 1]
```diff
@@ -494,7 +494,7 @@
     </data>
     
     <data name="W0111" xml:space="preserve">
-        <value>The iCloud entitlement '{item.Key}' is not supported by the Provisioning Profile.
+        <value>The iCloud entitlement '{0}' is not supported by the Provisioning Profile.
```
Comment (spouliot) [inline msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx:497]: was that not fixed in a different PR (maybe not on `main`) ?

**Review by spouliot — APPROVED** [review]

File: msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx (hunk 1 above)
Comment (rolfbjarne) [reply msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx:497]: Yes, I messed up when creating the PR.

Comment (rolfbjarne) [reply msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx]: It's fixed now.

**Comment by xamarin-release-manager** [comment]
✅ [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-9774/3/display/redirect) (on internal Jenkins) succeeded

✅ [Provisioning succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/3//console)
✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-9774/3//console)
✅ Packages built successfully
<details><summary>View packages</summary>

* [xamarin.ios-14.7.0.56.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/xamarin.ios-14.7.0.56.pkg)
* [Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Ref.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Runtime.ios-arm.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Runtime.ios-arm64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Runtime.ios-x64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Runtime.ios-x86.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.iOS.Sdk.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.macOS.Ref.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg)
* [Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.macOS.Runtime.osx-x64.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg)
* [Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.macOS.Sdk.10.15.100-ci.pr.gh9774.1179+sha.8f2cb9c01.nupkg)
* [Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.tvOS.Ref.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.tvOS.Runtime.tvos-arm64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.tvOS.Runtime.tvos-x64.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.tvOS.Sdk.14.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.watchOS.Ref.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.watchOS.Runtime.watchos-arm.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.watchOS.Runtime.watchos-x86.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)
* [Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/package/Microsoft.watchOS.Sdk.7.0.100-ci.pr.gh9774.555+sha.8f2cb9c01.nupkg)

</details>

✅ [API Diff (from stable)](http://xamarin-storage/jenkins/xamarin-macios/PR-9774/8f2cb9c0140c393f07082bb142d0f6d70a29c0a2/3/jenkins-results/api-diff/index.html)
🔥 [Failed to compare API and create generator diff](https://jenkins.internalx.com/job/macios/job/PR-9774/3//console) 🔥
&nbsp;&nbsp;&nbsp;&nbsp;Failed to build src/
&nbsp;&nbsp;&nbsp;&nbsp;Search for `Comparing API & creating generator diff` in the log to view the complete log.
ℹ️ Test run skipped: Not running tests here because they're run on public Jenkins.

**Review by chamons — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

**Comment by mandel-macaque** [comment]
we probably want somehow to backport this to xcode12 and xcode12.2

**Comment by rolfbjarne** [comment]
This makes the sample tests run at least, but it seems they're all failing due to something related to Xcode (which should be unrelated to this PR, so I'm merging this).

**Comment by rolfbjarne** [comment]
@monojenkins backport xcode12
@monojenkins backport xcode12.1

**Pull request merged**